.github/workflows/e2e.yml

```
@@ -0,0 +1,100 @@
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+          cache: false
+        id: go
+
+      - name: Install dependencies
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:9:30: Double quote to prevent globbing and word splitting

(shellcheck)